.github/workflows/build-docker-images.yml

```diff
@@ -70,16 +70,6 @@ jobs:
     name: "Latest PyTorch + DeepSpeed"
     runs-on: [intel-cpu, 8-cpu, ci]
     steps:
-      - name: Cleanup disk
-        run: |
-          sudo ls -l /usr/local/lib/
-          sudo ls -l /usr/share/
-          sudo du -sh /usr/local/lib/
-          sudo du -sh /usr/share/
-          sudo rm -rf /usr/local/lib/android
-          sudo rm -rf /usr/share/dotnet
-          sudo du -sh /usr/local/lib/
-          sudo du -sh /usr/share/
```

Comment:
we don't this any more as `[intel-cpu, 8-cpu, ci]` should be large enough.

Reply:
Does it hurt to keep?

Reply:
not really. But I prefer not to have it as it seems suggest we need it even if we are on `[intel-cpu, 8-cpu, ci]`. And if we don't have it in all places, it looks inconsistent and will be confusing IMO.